index.html

```diff
@@ -216,6 +216,49 @@ <h3>Skills</h3>
         </ul>
       </div>
     </section>
+     <section class="feedback">
+      <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+      <form action="#" method="post">
```

Comment:
Please make sure to integrate your `action` with [formspree service](https://formspree.io/html). visit the link and you can find the `API` endpoint then replace the endpoint with your `#` symbol. So that people can send you messages via this portfolio. Kindly implement it.